jetstream/outcomes/firefox_desktop/pocket_newtab.toml

```diff
@@ -1,72 +1,14 @@
 friendly_name = "Pocket in New Tab"
 description = "Usage and engagement metrics for Pocket content in New Tab."
 
-[metrics.pocket_organic_rec_clicks]
-friendly_name = "Pocket organic rec clicks"
-description = "Number of Pocket organic recs clicked in New Tab"
-select_expression = """
-      COUNTIF(
-          event = 'CLICK'
-          AND source = 'CARDGRID'
-          AND JSON_EXTRACT_SCALAR(value, '$.card_type') = 'organic'
-      )
-"""
-data_source = "as_events"
-statistics = { bootstrap_mean = {} }
+[metrics.organic_pocket_clicks.statistics.bootstrap_mean]
+[metrics.organic_pocket_clicks.statistics.deciles]
```

Comment:
This is cool! My assumptions is that this will test for differences in the distribution of Pocket clicks per client ID? How does partybal visualize such a thing?

Reply:
Yup, for differences in mean clicks per client. We'll get results on Experimenter that show confidence intervals for the means of the branches, as well as an interval for relative uplift in the metric due to the treatment. Partybal shows similar results, but it's where you'd have to look if segments were included -- Experimenter doesn't show results for segments yet.